### PR TITLE
Work on Create Tables and get Queries

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi There!

I have made the following changes to the repository:
- Create locally a database named `vet_clinic`. :card_file_box:
- Create table `animals` with project requirements ➕
- Insert data into `animals` ➕
- Query data from `animals` ⬇️
- Update `data.sql`, `queries.sql`, and `schema.sql` with all the information. 📝

## Here are the screenshots of the queries made to the table:
---
Captura de pantalla 2023-05-01 124933
---
Captura de pantalla 2023-05-01 125633
---
Captura de pantalla 2023-05-01 125638
---
Captura de pantalla 2023-05-01 125809
---
Captura de pantalla 2023-05-01 125847
---
Captura de pantalla 2023-05-01 125923
---
Captura de pantalla 2023-05-01 130018
---
Captura de pantalla 2023-05-01 132415
---

Thank you for taking the time to review my code! ⚡️🚀

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name, date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
+SELECT name, escape_attempts From animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```
Comment (mikethreels) [inline queries.sql]: - [x] according to the requirements you should also include a query for the following question:
  - list the name of all animals that are neutered and have less than 3 escape attempts.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Jaime, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great job creating and populating the table.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (jaimevillegas) [reply queries.sql]: Hi, @mikethreels!

Thank you for your comment! I worked on that suggestion, and it is already solved!

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @jaimevillegas👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**